lib/eventboss/queue_listener.rb

```diff
@@ -1,9 +1,34 @@
 module Eventboss
   class QueueListener
     class << self
+      def list_active
```

Comment:
I'm not huge fan of this name, how about `select(include:, exclude:)` (based on the comment below)?

Reply:
renamed to select